cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/TwoWindingsTransformerConversion.java

```diff
@@ -82,11 +89,33 @@ private void addTapChangers(TwoWindingsTransformer tx) {
         String ptc1 = end1.getId(ptcPropertyName);
         String ptc2 = end2.getId(ptcPropertyName);
 
+        // unused, will be delete in the full conversion
```

Comment:
If it is unused, I think you can delete it now.

Reply:
I changed the comment. It is used for debugging in current cgmes conversion. It will be not needed in the new cgmes conversion.